src/test/CMakeLists.txt

```diff
@@ -893,4 +893,9 @@ add_executable(unittest_any test_any.cc)
 add_ceph_unittest(unittest_any)
 target_link_libraries(unittest_any)
 
+# unittest_weighted_shuffle
+add_executable(unittest_weighted_shuffle test_weighted_shuffle.cc)
+add_ceph_unittest(unittest_weighted_shuffle)
+target_link_libraries(unittest_weighted_shuffle)
```

Comment:
if it does not link against any additional libraries, why bother adding this line?